js/index.js

```diff
@@ -0,0 +1,66 @@
+$(function() {
+	// variable to determine total number of guesses had - 5 guesses total
+	var totalGuesses = 5;
+	// declare variable that will be used to track the guess entered
+	var userGuess;
+	var computerChoice = getRandomIntInclusive();
+
+	$("#guessButton").click(calculate);
+	$("#reset").click(reset);
+
+		function getRandomIntInclusive(min, max) {
+		// Returns a random integer between min (included) and max (included)
+		  return Math.floor(Math.random() * (100 - 1 + 1)) + 1;
+		  console.log(getRandomIntInclusive);
```

Comment:
@humphriescarly Just so you know, the `return` statement in JavaScript does 2 main things:

1. "spits out" a value, in your case a random number, from a function
2. immediately stops the function from running any further

So just so you know, this console log statement here will _never_ get hit since its _below_ the return statement and return exits the function immediately. No code below a return will be executed.